src/attributes.jl

```diff
@@ -855,15 +855,24 @@ The values indicate how to interpret the result vector.
   infeasibility.
 * `NEARLY_INFEASIBILITY_CERTIFICATE`: the result satisfies a relaxed criterion for
   a certificate of infeasibility.
+* `REDUCTION_CERTIFICATE`: the result vector is an ill-posed certificate; see
+  [this article](https://doi.org/10.1007/s10107-017-1169-9) for details. If the
```

Comment:
It's better to link to the freely available version: https://arxiv.org/abs/1408.4685.